server/src/modules/db/constant.js

```diff
@@ -0,0 +1,26 @@
+
+const PUBLIC_VISIBILITY = 'Public';
+const PRIVATE_VISIBILITY = 'Private';
+const UNLISTED_VISIBILITY = 'Unlisted';
+
+const VIDEO_VISIBILITIES = [PUBLIC_VISIBILITY, PRIVATE_VISIBILITY, UNLISTED_VISIBILITY];
```

Comment:
try like below:

```javascript
const VIDEO_VISIBILITIES = {
PUBLIC : 'public',
PRIVATE : 'private'
}
```

and use like
`const someVariable = VIDEO_VISIBILITIES.PUBLIC`

Reply:
and delete these lines coz we don't need these
```
const PUBLIC_VISIBILITY = 'Public';
const PRIVATE_VISIBILITY = 'Private';
const UNLISTED_VISIBILITY = 'Unlisted';
```

Reply:
> try like below:
> 
> ```js
> const VIDEO_VISIBILITIES = {
> PUBLIC : 'public',
> PRIVATE : 'private'
> }
> ```
> 
> and use like `const someVariable = VIDEO_VISIBILITIES.PUBLIC`

vai, i had the same idea, but we also pass `VIDEO_VISIBILITIES` for the video seed data and `faker.helpers.arrayElement` expect an array.

Reply:
> > try like below:
> > ```js
> > const VIDEO_VISIBILITIES = {
> > PUBLIC : 'public',
> > PRIVATE : 'private'
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > and use like `const someVariable = VIDEO_VISIBILITIES.PUBLIC`
> 
> vai, i had the same idea, but we also pass `VIDEO_VISIBILITIES` for the video seed data and `faker.helpers.arrayElement` expect an array.

using `Object.values(VIDEO_VISIBILITIES)` for getting a array with all `VIDEO_VISIBILITIES`  values